e2e-tests/src/test/utility.rs

```diff
@@ -11,25 +8,15 @@ pub fn batch_transactions(config: &Config) -> anyhow::Result<()> {
 
     let (connection, to) = setup_for_transfer(config);
 
-    let call = compose_call!(
-        connection.as_connection().metadata,
-        "Balances",
-        "transfer",
-        GenericAddress::Id(to),
-        Compact(1000u128)
-    );
+    let call = connection.create_transfer_tx(to, 1000u128);
     let mut transactions = Vec::new();
     for _i in 0..NUMBER_OF_TRANSACTIONS {
         transactions.push(call.clone());
     }
 
-    let extrinsic =
```

Comment:
This can we extract to new `utility` module in https://github.com/Cardinal-Cryptography/aleph-node/tree/main/aleph-client/src